flink-connectors/flink-connector-jdbc/pom.xml

```diff
@@ -68,6 +68,14 @@ under the License.
 			<scope>provided</scope>
 		</dependency>
 
+		<!-- SQL Server -->
+		<dependency>
+			<groupId>com.microsoft.sqlserver</groupId>
+			<artifactId>mssql-jdbc</artifactId>
+			<version>10.2.0.jre8</version>
```

Comment:
Could the version be the latest version ?

Reply:
Hi, @RocMarshal , are you referring to v11.x.x? From my understanding, there are still at preview release, at 10.2.x is the latest stable release. SHould we use v11?

https://github.com/microsoft/mssql-jdbc/releases/tag/v.11.1.0

I've bumped it to 10.2.1 though

Reply:
Okay~.  Just keep the current version well.